pipeline/02-assess.R

```diff
@@ -314,18 +312,22 @@ message("Saving card-level data")
 
 # Keep only card-level variables of interest, including: ID variables (run_id,
 # pin, card), characteristics, and predictions
+char_vars <- params$model$predictor$all[
+  grepl("^char_", params$model$predictor$all)
+]
+char_vars <- char_vars[!char_vars %in% c("char_apts", "char_recent_renovation")]
```

Comment:
**[Question, non-blocking]** Hmm, does this mean we should have also adjusted encoding of `char_recent_renovation` in https://github.com/ccao-data/model-res-avm/pull/210?

Reply:
No you're all good, it shouldn't change the outcome there because I don't think it's in the variable dictionary.